src/bin/cargo/commands/install.rs

```diff
@@ -16,6 +16,7 @@ use cargo_util::paths;
 pub fn cli() -> Command {
     subcommand("install")
         .about("Install a Rust binary")
+        .arg_dry_run("Perform all checks without installing (unstable)")
```

Comment:
> Tests and documentation done in https://github.com/rust-lang/cargo/pull/13598, may be cherry picked into this PR if needed.

This would be expected for merging

Reply:
I added the doc made in #13598 and removed the part about the specific dry run messages